enterprise/frontend/src/metabase-enterprise/whitelabel/components/LighthouseToggleWidget/LighthouseToggleWidget.styled.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from "@emotion/styled";
+import { hueRotate } from "metabase/lib/colors";
+
+export const LighthouseImage = styled.div`
+  width: 99.5px;
```

Comment:
Why not `100px`? Seeing a fraction seems a bit strange. I believed this is from down-sizing the Metabot image which is 1194 × 1080px and is scaled down to 99.5 x 90px, but we could use a different number for the lighthouse.